apps/src/applab/applab.js

```diff
@@ -845,6 +845,10 @@ studioApp.runButtonClick = function() {
   studioApp.attempts++;
   Applab.execute();
 
+  // Show view data button now that channel id is available.
```

Comment:
I assume that channel id isn't available until you receive an async response from the dashboard? Can the button be enabled when the async response is received?